### ISSUE TITLE
Self Registration link is showing "Unauthorized" for Gitllab access. #Functional Bug

### ISSUE THREAD

**Issue by AnshuBhatt** [topcoder-platform/Training-Challenges#22]
**Steps to Reproduce:** 

You need to follow the below steps if you are using first-time GitLab or the self-registration link:
1. Create a GitLab account if you do not have already.
2. Login to Topcoder.
3. Use the self-registration link and register yourself - https://x.topcoder.com/api/v1/gitlab/groups/registration/3c718072-0775-4487-865b-801e1da4ff99-1547477444703
4. The user is redirected to authorize the page on gitlab stating "Authorize Topcoder-X to use your account?"
5. Click on "Authorize"
6. The user is redirected with the message "Unauthorized".

**Actual:**
The self-registration link is not working properly.

**Expected:**
The user is expected to access the Gitlab repo using the self-registration link(of Topcoder).

**Device/OS/Browser Information:**
Windows 10 chrome latest

**Screenshot**
![Screenshot (1299)](https://user-images.githubusercontent.com/38535720/69899797-7c035400-1391-11ea-9ecb-6be0cee83796.png)
![Screenshot (1298)](https://user-images.githubusercontent.com/38535720/69899798-7c035400-1391-11ea-8e9c-047b1b84669f.png)

**Comment by nithyaasworld** [comment]
@AnshuBhatt Can you tell me where did you get the link you have provided in the step:3 above from?

**Comment by AnshuBhatt** [comment]
Hello Mam,

I registered for a challenge(Test for Success Challenge Series - QA
Challenge 50 (Bug Hunt) v1.0") and in the forum(https://apps.topcoder.
com/forums/?module=Message&messageID=2378711) there was a link for self
registration for gitlab account to take part in QA challenges. So, I
clicked on that link but I showed me Message:Unauthorized. I asked in the
forum also about this issue and another person was also facing the same
issue. So meantime, I took part in this challenge. And, I reported this
issue there.

And due to this issue, I am unable to access the gitlab repo. And, hence
unable to understand the challenge details and information provided there.
Kindly ask the team to fix this issue as fast as possible.
